client/model.go

```diff
@@ -1225,3 +1225,13 @@ func (e *AuditLogEntry) DecisionLog() (*auditv1.DecisionLogEntry, error) {
 type PlanResourcesResponse struct {
 	*responsev1.PlanResourcesResponse
 }
+
+type (
+	ListPoliciesOption func(*requestv1.ListPoliciesRequest)
+)
+
+func WithIncludeDisabled(value bool) ListPoliciesOption {
+	return func(request *requestv1.ListPoliciesRequest) {
+		request.IncludeDisabled = value
+	}
+}
```

Comment:
Sorry I missed this earlier. We don't need the boolean argument here because it's false by default anyway. Just name it `WithIncludeDisabled()`.